gwas-rabbitmq-listener/src/main/java/uk/ac/ebi/spot/gwas/rabbitmq/consumer/PublicationRabbitConsumer.java

```diff
@@ -0,0 +1,40 @@
+package uk.ac.ebi.spot.gwas.rabbitmq.consumer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.ac.ebi.spot.gwas.deposition.dto.curation.PublicationRabbitMessage;
+import uk.ac.ebi.spot.gwas.rabbitmq.constants.DepositionCurationConstants;
+import uk.ac.ebi.spot.gwas.rabbitmq.service.PublicationImportService;
+
+/**
```

Comment:
@sajo-ebi could we just annotate this component with @Profile(value = { "test"}) which will help us restrict its execution only to the test profile environment. 

And we should do thesame for this (https://github.com/EBISPOT/gwas-curation-service/blob/develop/src/main/java/uk/ac/ebi/spot/gwas/curation/rabbitmq/MetadataYmlUpdateConsumer.java) which was commentedn out in the MetadataYaml implementation.

This will save us from commenting and uncommenting.